plugins/CorePluginsAdmin/tests/UI/TagManagerTeaser_spec.js

```diff
@@ -82,11 +82,11 @@ describe("TagManagerTeaser", function () {
     });
 
     it('should be possible to disable page and redirect to home', function (done) {
-        capturePage(done, 'admin_page_disable', function (page) {
+        expect.page("").contains("#dashboard", function (page) {
```

Comment:
Is that change on purpose?

Reply:
there is no screenshot name etc... and shouldn't it capture?

Reply:
yes. it now only checks if the loaded page has the element `#dashboard`. Imho there is no need to compare the actual content as a screenshot